core/src/main/java/org/kiwix/kiwixmobile/core/main/AddNoteDialog.java

```diff
@@ -238,9 +239,11 @@ private void exitAddNoteDialog() {
         .findFragmentByTag(ConfirmationAlertDialogFragment.TAG);
 
       if (previousInstance == null) {
+        AlertDialogShower alertDialogShower = new AlertDialogShower(getActivity());
```

Comment:
Must be instantiated by Dagger

Reply:
@macgills  The app is not building when I am adding the injection line i.e.
`  @Inject protected AlertDialogShower alertDialogShower;
`
Its throwing this error 

> D:\kiwix-android\core\src\main\java\org\kiwix\kiwixmobile\core\data\local\dao\BookDao.java:43: error: cannot find symbol
  private void setBookDetails(Book book, SquidCursor<BookDatabaseEntity> bookCursor) {
                                                     ^
  symbol:   class BookDatabaseEntity
  location: class BookDaoD:\kiwix-android\core\src\main\java\org\kiwix\kiwixmobile\core\data\local\dao\BookmarksDao.java:42: error: cannot find symbol
  public List<Bookmark> getBookmarks() {
              ^
  symbol:   class Bookmark
  location: class BookmarksDaoD:\kiwix-android\core\src\main\java\org\kiwix\kiwixmobile\core\data\local\dao\RecentSearchDao.java:45: error: cannot find symbol
  public List<RecentSearch> getRecentSearches() {
              ^
  symbol:   class RecentSearch


Could you please help to find what am I doing wrong?